lago/vm.py

```diff
@@ -144,7 +144,7 @@ def shutdown(self, *args, **kwargs):
         )
 
         try:
-            with utils.ExceptionTimer(timeout=15):
+            with utils.ExceptionTimer(timeout=60 * 5):
                 while self.defined():
                     time.sleep(0.1)
```

Comment:
Are you sure you are not nagging the libvirt too often? I think once a second is enough.